src/Application.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-cli for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-cli/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-cli/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Laminas\Cli;
+
+use Symfony\Component\Console\Application as SymfonyConsoleApplication;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @internal
+ */
+final class Application extends SymfonyConsoleApplication
```

Comment:
I see it's marked internal, but _why_ is the class needed? What purpose does it serve for the package?

Reply:
So, make a comment indicating some of the above - that it's to facilitate access to the input and output for params. That way we won't have somebody suggesting we remove it!